GameAssets/bannerIntro.txt

```diff
@@ -0,0 +1,20 @@
+*************************************************************************************************************
+
+  .oooooo.   oooo                               o8o                 ooooooooo.   ooooooooo.     .oooooo.    
+ d8P'  `Y8b  `888                               `"'                 `888   `Y88. `888   `Y88.  d8P'  `Y8b   
```

Comment:
I really like the idea of having a startup screen. I think this will really help the game feel more immersive, thanks for submitting this!